misk-actions/src/main/kotlin/misk/MiskCaller.kt

```diff
@@ -29,6 +29,7 @@ data class MiskCaller @JvmOverloads constructor(
   }
 
   /** Determine based on allowed capabilities/services if the caller is permitted */
+  @Deprecated("This has been inlined into AccessInterceptor.isAuthorized()")
```

Comment:
why was this deprecated? what are services supposed to change to using?

Reply:
My belief is that only AccessInterceptor or re-implementations of AccessInterceptor would want to use it.